style.css

```diff
@@ -1,2 +1,199 @@
-body { font-family: "Montserrat", serif; }
+body {
+  background: #fff;
+  font-family: "Montserrat", serif;
+  height: 2700px;
+  width: 1440px;
+}
 
+.wordmarksmall {
+  /* height: 51px; 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/IFW/pullRequest?prid=6044074)